### PR TITLE
fix: fix tmpfs_size and tmpfs_mode not being set

### PR DESCRIPTION
**Pull request by kristof-mattei**
##### SUMMARY

Fixes a bug where `tmpfs_size` and `tmpfs_mode` weren't correctly set on the Docker Container.

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

Docker Container - parsing of `tmpfs_size` and `tmpfs_mode`

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: tests/integration/targets/docker_swarm_service/tasks/tests/mounts.yml [hunk 1]
```diff
@@ -418,7 +418,6 @@
         type: "tmpfs"
         tmpfs_size: "50M"
   register: mounts_tmpfs_size_1
-  ignore_errors: yes
```
Comment (felixfontein) [inline tests/integration/targets/docker_swarm_service/tasks/tests/mounts.yml:421]: Please keep these, and modify the conditions in the assertions below instead.

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution! Can you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment)? Thanks!

File: tests/integration/targets/docker_swarm_service/tasks/tests/mounts.yml (hunk 1 above)
Comment (kristof-mattei) [reply tests/integration/targets/docker_swarm_service/tasks/tests/mounts.yml]: You're right. Got rid of it.

File: changelogs/fragments/fix-tmpfs_size-and-tmpfs_mode.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - docker_container - fix tmfs_size and tmpfs_mode not being set
```
Comment (felixfontein) [inline changelogs/fragments/fix-tmpfs_size-and-tmpfs_mode.yml:2]:
```suggestion
  - docker_container - fix ``tmfs_size`` and ``tmpfs_mode`` not being set (https://github.com/ansible-collections/community.docker/pull/580).
```

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@kristof-mattei thanks a lot for fixing this!

**Pull request merged**